spec/features/idv/in_person_threatmetrix_spec.rb

```diff
@@ -249,39 +249,37 @@ def deactivate_profile_update_enrollment(status:)
         end
       end
 
-      # To be completed in a future ticket
-      # context 'User fails IPP and fails TMX review' do
-      #   it_behaves_like 'initially shows the user the barcode page'
-
-      #   it 'allows the user to restart the flow', allow_browser_log: true do
-      #     deactivate_profile_update_enrollment(status: :failed)
-
-      #     visit_idp_from_sp_with_ial2(sp)
-      #     # user should not see the please call page as they failed ipp
-      #     expect(page).not_to have_current_path(idv_please_call_path)
-      #     # redo the flow:
-      #     complete_entire_ipp_flow(user, tmx_status)
-      #     expect(page).to have_current_path(idv_in_person_ready_to_verify_path)
-      #   end
-      # end
-
-      # To be completed in a future ticket
-      # context 'User cancels IPP after being deactivated for TMX review',
-      #         allow_browser_log: true do
-      #   it 'allows the user to restart IPP' do
-      #     # set fraud review to pending:
-      #     profile.deactivate_for_fraud_review
-      #     profile.reload
-      #
-      #     # cancel the enrollment
-      #     click_link t('links.cancel')
-      #     # user can restart
-      #     click_on t('idv.cancel.actions.start_over')
-      #
-      #     # user should be redirected to the welcome path when they cancel
-      #     expect(page).to have_current_path(idv_welcome_path)
-      #   end
-      # end
+      context 'User fails IPP and fails TMX review' do
+        it_behaves_like 'initially shows the user the barcode page'
```

Comment:
Hmm... I am not sure I like this. If the user failed IPP they would never see that barcode page, right? I feel like this is testing a state a user should never be in. What do you think?

Reply:
Well I think this is correct because the user needs to receive a barcode to take to the post office. The `fails IPP` bit is referring to the user failing proofing at the post office. Let me know if I can make that clearer in the spec. Basically all of the interesting stuff we do with TMX happens after the user has visited the post office.

Reply:
Oh, I can see how that's misleading.

We're using it to do the initial setup: the user initially sees the barcode page, then (further down in the test case) goes to the Post Office and fails IPP.

Reply:
I goofed. I updated my comment above while/as you posted. I was thinking it was the tmx value- not Team Judy. Sorry about that.